contents/handbook/company/team/paolo-damico.md

```diff
@@ -1,22 +1,35 @@
 sidebar: Handbook
 showTitle: true
 ---
 
-## Goals (6 weeks until 6 Aug 2021)
+## Q4 2021 Goals
 
-*   **Execute on nailing funnels.** Ensure the funnels product actually solves user’s needs and is shipped with the highest user experience quality.
-    * Ensure engineers have all the context needed to Nail Funnels, particularly information around user’s problems / use cases, user feedback, and relevant data insights.
-    * Support engineers in scoping problems so they maximize for value delivered and can be built in a constrained time frame.
-    * Provide feedback, test features thoroughly and report/fix issues based on dogfooding, user feedback and usability tests to ensure a high quality experience.
-    * Support Marcus and the entire team measure success through qualitative, quantitative metric(s).
-*   **On what’s next.** Provide as much context as possible (mainly on user input, usage data, and user experience intuition) for engineers and the Exec Team to decide what to work on next (on an operational and strategic level, respectively).
-    * Support engineers set up their own behavioral metrics and encourage them to measure results on their own work and figure out improvements.
-    * Provide context from user interviews, data analyses and other sources to figure out sprint priorities. Includes helping break down problems.
-    * Provide internal product context to non-engineering members of the team as we figure out what to focus on next. 
-*   **Engineering**
-    *   Ship series/steps naming feature OR saved reports.
-    *   Ship UI/UX improvements on funnels as features are built.
-    *   Pick up any critical bug or UX issue if no one else can.
+- **Product strategy.**
+  - ⭐️ Build the _core product_ strategy for Q4-21 (particularly related to Collaboration & Experimentation).
+  - Support Exec Team with context, feedback and any other activity in shaping up the product strategy for the next quarter/year. In particular, ensure we have clear company-level metrics and we have a plan for improving them.
+- **Product execution.**
+  - ⭐️ Ensure ownership and accountability on every key feature by a single engineer, particularly with defining clear and ambitious goals for each epic in each sprint. *Help owners improve on workflow, prioritization, solution scoping (particularly around overscoping) & post-delivery.*
+  - Clearly define what success is for each milestone.
+  - Successful (*see above*) delivery of all milestones from product strategy.
+- **User experience.**
+  - ⭐️ Develop a strong culture within Product & Engineering for the quality bar of features. Improve the bar for core shipped features, lower the bar for MVPs.
+  - Reduce my number of **code** reviews and PRs (*to increase the depth of UX reviews*).
+- **Enterprise & Growth support.**
+  - Ensure the essential set of 1) enterprise and 2) revenue features are built in a timely fashion (including prioritization, scoping, context sharing) to unblock customers.
```

Comment:
Yeah I think the balance is important here...

We never want to be blocking enterprise sales - but at the same time we also don't want to be building stuff before they need it and using up our resources.